ultra/ultra/scenarios/generate_scenarios.py

```diff
@@ -437,7 +437,17 @@ def generate_left_turn_missions(
         speed_m_per_s = float("".join(filter(str.isdigit, speed))) * 5.0 / 18.0
         hijacking_params = route_distributions["ego_hijacking_params"]
         zone_range = hijacking_params["zone_range"]
-        waiting_time = hijacking_params["wait_to_hijack_limit_s"]
+
```

Comment:
Wait sorry I might have missed why we need this change, is there something it could help with?

Reply:
Actually, this change is not very significant in the grand scheme of generating scenarios. Currently, the only way that we can change the ego vehicle starting "time" is by altering the hijack limit inside the `distributions.py`.  This feature will allow users to set a range to when can the ego vehicle be hijacked. It is not necessary for this feature to be added.